python/ray/autoscaler/v2/instance_manager/config.py

```diff
@@ -128,6 +128,8 @@ class NodeTypeConfig:
     min_worker_nodes: int
     # The maximal number of worker nodes can be launched for this node type.
     max_worker_nodes: int
+    # Idle timeout seconds for worker nodes of this node type.
+    idle_timeout_s: Optional[int] = None
```

Comment:
Keep it consistent with the Ray codebase: https://github.com/ray-project/ray/blob/154915d3f3e2d65fd7dd7c840ad8f77959bf54b8/python/ray/autoscaler/v2/scheduler.py#L653

```suggestion
    idle_timeout_s: Optional[float] = None
```